CMakeGlobals.txt

```diff
@@ -145,6 +145,9 @@ endif()
 # set session32 if specified
 if(RSTUDIO_TARGET STREQUAL "SessionWin32")
    set(RSTUDIO_SESSION_WIN32 TRUE)
+   add_definitions(-D_X86_)
```

Comment:
This seems to be needed to ensure we compile for the right architecture when running on ARM64 Windows.